### PR TITLE
Adding wizards to command options

### PR DESCRIPTION
**Pull request by lucasncabral**
In this PR we added a wizard to help novice users run some commands:

```
$ ml-git datasets create dataset-ex
At least one category must be defined. The categories names must be comma-separated. Please inform the value for categories option: test, category
An entity must have a defined mutability. Please inform the value for mutability option (strict, flexible, mutable): strict
INFO - MLGit: Dataset artifact created.
```

```
$ ml-git repository storage add storage-ex
Define the storage type (s3h, azureblobh, gdriveh, sftph) [s3h]: s3h
Define the profile to be used to connect to S3 [default]: profile
If you are using MinIO define the endpoint URL [None]:
Define the AWS region name for the storage [us-east-1]: us-east-2
INFO - Admin: Add storage [s3h://storage-ex] with creds from profile [profile]
INFO - Admin: When making changes to the config file we strongly recommend that you upload these changes to the Git repository. For this, see: ml-git repository config push --help
```

```
$ ml-git models add model-ex
If you have a file with the metrics of this model you can associate it with this version of the entity, define the path to the file [None]:
INFO - Metadata Manager: Pull [/home/dev/demo/.ml-git/datasets/metadata]                                                                                                                         
INFO - Repository: datasets adding path [/home/dev/demo/datasets/dataset-ex] to ml-git index                                                                                                     
files: 100%|██████████████████████████████████████| 1.00/1.00 [00:00<00:00, 294files/s]
files: 100%|██████████████████████████████████████| 1.00/1.00 [00:00<00:00, 3.84kfiles/s]
```

```
$ ml-git models commit model-ex
Link dataset entity name to this model set version [empty for none]:
Link label entity name to this model set version [empty for none]:
INFO - Metadata Manager: Commit repo[/home/dev/demo/.ml-git/datasets/metadata] --- file[computer-vision/images/dataset-ex]
```

### CODE REVIEW

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -1043,7 +1045,8 @@ ml-git repository remote config add https://git@github.com/mlgit-config
 ```
 Usage: ml-git repository storage add [OPTIONS] BUCKET_NAME
 
-  Add a storage BUCKET_NAME to ml-git
+  Add a storage BUCKET_NAME to ml-git [This command has a wizard that 
+  will request the necessary information if they are not informed
```
Comment (Vieira-Carlos) [inline docs/mlgit_commands.md:1049]:
```suggestion
  will request the necessary information if it is not informed
```

File: ml_git/commands/help_msg.py [hunk 2]
```diff
@@ -79,3 +79,8 @@
 VERBOSE_OPTION = 'Debug mode'
 FSCK_FULL_OPTION = 'Show the list of corrupted files.'
 REMOTE_FSCK_FULL_OPTION = 'Show the list of fixed and unfixed blobs and IPLDs.'
+CREATE_COMMAND = 'This command will create the workspace structure with data and spec file for an entity and set the ' \
+                 'git and storage configurations. [This command has a wizard that will request the necessary information ' \
+                 'if they are not informed]'
```
Comment (Vieira-Carlos) [inline ml_git/commands/help_msg.py:84]:
```suggestion
                 'if it is not informed]'
```

File: ml_git/commands/help_msg.py [hunk 3]
```diff
@@ -79,3 +79,8 @@
...
+STORAGE_ADD_COMMAND = 'Add a storage BUCKET_NAME to ml-git. [This command has a wizard that will request the necessary' \
+                      ' information if they are not informed'
```
Comment (Vieira-Carlos) [inline ml_git/commands/help_msg.py:86]:
```suggestion
                      ' information if it is not informed'
```

File: ml_git/commands/prompt_msg.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+"""
+© Copyright 2022 HP Development Company, L.P.
+SPDX-License-Identifier: GPL-2.0-only
+"""
+
+CATEGORIES_MESSAGE = 'At least one category must be defined. The categories names must be comma-separated. ' \
+                     'Please inform the value for categories option'
```
Comment (Vieira-Carlos) [inline ml_git/commands/prompt_msg.py:7]:
```suggestion
                     'Please inform the value for the categories option'
```

File: ml_git/commands/prompt_msg.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+LINKED_DATASET_TO_MODEL_MESSAGE = 'Link dataset entity name to this model set version'
+LINKED_LABEL_TO_MODEL_MESSAGE = 'Link label entity name to this model set version'
+LINKED_DATASET_TO_LABEL_MESSAGE = 'Link dataset entity name to this label set version'
+MUTABILITY_MESSAGE = 'An entity must have a defined mutability. Please inform the value for mutability option'
```
Comment (Vieira-Carlos) [inline ml_git/commands/prompt_msg.py:11]:
```suggestion
MUTABILITY_MESSAGE = 'An entity must have a defined mutability. Please inform the value for the mutability option'
```

**Review by Vieira-Carlos — CHANGES_REQUESTED** [review]
Just some typos

File: docs/mlgit_commands.md (hunk 1 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: The suggestion was applied in 7c084b7.

File: ml_git/commands/help_msg.py (hunk 3 above)
Comment (lucasncabral) [reply ml_git/commands/help_msg.py]: The suggestion was applied in 7c084b7.

File: ml_git/commands/prompt_msg.py (hunk 5 above)
Comment (lucasncabral) [reply ml_git/commands/prompt_msg.py]: The suggestion was applied in 7c084b7.

File: ml_git/commands/prompt_msg.py (hunk 4 above)
Comment (lucasncabral) [reply ml_git/commands/prompt_msg.py]: The suggestion was applied in 7c084b7.

File: ml_git/commands/help_msg.py (hunk 2 above)
Comment (lucasncabral) [reply ml_git/commands/help_msg.py]: The suggestion was applied in 7c084b7.

**Review by Vieira-Carlos — APPROVED** [review]

File: docs/mlgit_commands.md [hunk 6]
```diff
@@ -1043,7 +1045,8 @@ ml-git repository remote config add https://git@github.com/mlgit-config
 ```
 Usage: ml-git repository storage add [OPTIONS] BUCKET_NAME
 
-  Add a storage BUCKET_NAME to ml-git
+  Add a storage BUCKET_NAME to ml-git [This command has a wizard that 
+  will request the necessary information if it is not informed
```
Comment (andersonpablosilva) [inline docs/mlgit_commands.md:1049]:
```suggestion
  will request the necessary information if it is not informed]
```

File: ml_git/commands/help_msg.py [hunk 7]
```diff
@@ -79,3 +79,8 @@
 VERBOSE_OPTION = 'Debug mode'
 FSCK_FULL_OPTION = 'Show the list of corrupted files.'
 REMOTE_FSCK_FULL_OPTION = 'Show the list of fixed and unfixed blobs and IPLDs.'
+CREATE_COMMAND = 'This command will create the workspace structure with data and spec file for an entity and set the ' \
+                 'git and storage configurations. [This command has a wizard that will request the necessary information ' \
+                 'if it is not informed]'
+STORAGE_ADD_COMMAND = 'Add a storage BUCKET_NAME to ml-git. [This command has a wizard that will request the necessary' \
+                      ' information if it is not informed'
```
Comment (andersonpablosilva) [inline ml_git/commands/help_msg.py:86]:
```suggestion
                      ' information if it is not informed]'
```

File: ml_git/commands/prompt_msg.py [hunk 8]
```diff
@@ -0,0 +1,22 @@
+"""
+© Copyright 2022 HP Development Company, L.P.
+SPDX-License-Identifier: GPL-2.0-only
+"""
+
+CATEGORIES_MESSAGE = 'At least one category must be defined. The categories names must be comma-separated. ' \
```
Comment (andersonpablosilva) [inline ml_git/commands/prompt_msg.py:6]:
```suggestion
CATEGORIES_MESSAGE = 'At least one category must be defined. The categories' names must be comma-separated. ' \
```

File: ml_git/commands/prompt_msg.py [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+LINKED_LABEL_TO_MODEL_MESSAGE = 'Link label entity name to this model set version'
+LINKED_DATASET_TO_LABEL_MESSAGE = 'Link dataset entity name to this label set version'
+MUTABILITY_MESSAGE = 'An entity must have a defined mutability. Please inform the value for the mutability option'
+EMPTY_FOR_NONE = 'empty for none'
+STORAGE_TYPE_MESSAGE = 'Define the storage type'
+CREDENTIALS_PROFILE_MESSAGE = 'Define the profile to be used to connect to S3 [default]'
+CREDENTIALS_PATH_MESSAGE = 'Define the path to the credentials that will be used to connect to GDrive [\'.\']'
+REGION_MESSAGE = 'Define the AWS region name for the storage [us-east-1]'
+ENDPOINT_MESSAGE = 'If you are using MinIO define the endpoint URL [None]'
+USERNAME_SFTPH = 'Define the username for the sftp login [None]'
```
Comment (andersonpablosilva) [inline ml_git/commands/prompt_msg.py:18]:
```suggestion
USERNAME_SFTPH = 'Define the username for the SFTP login [None]'
```

**Review by andersonpablosilva — COMMENTED** [review]
Just a few improvements.

File: docs/mlgit_commands.md (hunk 6 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: The suggestion was applied in 1c1ec60.

File: ml_git/commands/help_msg.py (hunk 7 above)
Comment (lucasncabral) [reply ml_git/commands/help_msg.py]: The suggestion was applied in 1c1ec60.

File: ml_git/commands/prompt_msg.py (hunk 8 above)
Comment (lucasncabral) [reply ml_git/commands/prompt_msg.py]: The suggestion was applied in https://github.com/HPInc/ml-git/commit/1c1ec60bb5cf6c27227f1d13717eaae70482550e.

File: ml_git/commands/prompt_msg.py (hunk 9 above)
Comment (lucasncabral) [reply ml_git/commands/prompt_msg.py]: The suggestion was applied in https://github.com/HPInc/ml-git/commit/1c1ec60bb5cf6c27227f1d13717eaae70482550e.

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**